core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -253,6 +253,13 @@ trait DeltaSQLConfBase {
       .checkValue(_ > 0, "parallelDelete.parallelism must be positive")
       .createOptional
 
+  val DELTA_VACUUM_FILE_LISTING_PARALLELISM =
+    buildConf("vacuum.fileListing.parallelism")
+      .doc("Sets the number of partitions to use for file listing")
+      .intConf
+      .checkValue(_ > 0, "fileListing.parallelism must be positive")
+      .createWithDefault(200)
```

Comment:
should we set a default here ? or do you think defaulting to spark.sql.shuffle.partitions is more ideal ?

Reply:
good point, I even think that defaulting to spark.default.parallelism is more ideal.

Reply:
@rahulsmahadev  @vkorukanti any comments on this PR?